stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/util/StorageHelperWrapper.kt

```diff
@@ -29,25 +30,32 @@ import io.getstream.chat.android.client.models.Attachment
  * @param context The context of the app, used to fetch files and media.
  * @param storageHelper The storage helper that provides all the logic required to work with the
  * system storage.
+ * @param attachmentFilter A filter that is used to filter out attachments that will not be accepted
+ * by the backend.
  */
 public class StorageHelperWrapper(
     private val context: Context,
     private val storageHelper: StorageHelper,
+    private val attachmentFilter: AttachmentFilter,
 ) {
 
     /**
      * Loads a list of file metadata from the system.
```

Comment:
Maybe we just need to note in the kdocs for these methods that we filter the list against accepted mime types, but I'd leave this one up to your judgement.

Maybe something like:

```suggestion
     * Loads a list of file metadata from the system and filters it against file types accepted by the backend.
```